src/action.ts

```diff
@@ -44,29 +44,33 @@ async function run() {
         }
 
         if (process.env.GITHUB_TOKEN) {
-            const octokit = github.getOctokit(process.env.GITHUB_TOKEN);
-            const { repoLatest, majorLatest } = await findLatestReleases(octokit);
+            core.info(
+                "Using obsolete GITHUB_TOKEN environment variable, please set an input
+                value instead. In most cases the default value will just work and you can
+                simply remove the token variable from your configuration.”
+            );
+        }
+        const token = core.gitInput('token')
```

Comment:
```suggestion
        const token = core.getInput('token')
```

Reply:
There’s also a `required` option if you want it to throw if the option is missing.